package com.moulberry.flashback.playback;

import net.minecraft.network.Connection;
import net.minecraft.server.MinecraftServer;
import net.minecraft.server.level.ServerPlayer;
import net.minecraft.server.network.ServerGamePacketListenerImpl;

public class FlashbackFakePlayerPacketListener extends ServerGamePacketListenerImpl {

    public FlashbackFakePlayerPacketListener(MinecraftServer server, Connection connection, ServerPlayer player) {
        super(server, connection, player);
    }

}
